content/about/faq.md

```diff
@@ -51,27 +51,23 @@ Canvas Panel works as a user interface to render IIIF deep-zooming images. It al
 
 ### What is our timeline for transitioning to 11ty?
 
-Work on 11ty is still underway. We hope to complete the transition by July 2022. We look to the Quire community to help us spot any bugs that the switch from Hugo to 11ty may have generated. The move will be a gradual work in progress, and we appreciate our users' feedback and patience.
+We will launch a pre-release version of Quire in October 2022 that will be fully open source. We look to the community to test out new projects and help us spot bugs that the switch from Hugo to 11ty may have generated. After major issues have been addressed, a 1.0 release of Quire will follow. We appreciate our users' feedback and patience during this transition period.
```

Comment:
> We will launch a pre-release version of Quire in October 2022 that will be fully open source.

Did we have a conversation about making the pre-release officially open source?

Reply:
Yes. Or at least we started to discuss it. I mentioned that if we were going to indefinitely delay the v 1.0 launch then it would be best if we could at least announce that we are fully OS with the pre-release since that seems to be the thing *several* people are waiting on. It also would be good for messaging since we've been pushing back the "official OS launch" since last spring. I thought both you and Matthew said you didn't see an issue with it but maybe I misunderstood. I'm fine with it only really being announced to our community of users and maybe changing some language on our website, etc. No big fan fare until the v 1.0 launch.

Reply:
@geealbers let me know if there are any additional changes I can/should make so this can be merged.

Reply:
@Erin-Cecele you can go ahead and merge. thanks